Makefile

```diff
@@ -92,6 +94,21 @@ TRIVY_VERSION ?= 0.14.0
 PROTOC_VERSION ?= 3.15.2
 SHELLCHECK_VER ?= v0.7.2
 
+
+# For aws integration tests 
+BUILD_TIMESTAMP_W_SEC := $(shell date +%Y-%m-%d-%H-%M-%S)
+EKS_CLUSTER_NAME := integ-cluster-$(BUILD_TIMESTAMP_W_SEC)
+AWS_SERVICE_ACCOUNT_NAME := basic-test-mount-sa
+
+AWS_REGION := us-west-2
```

Comment:
Do we need to randomize the region or are we certain `us-west-2` will always have the required capacity to run the aws tests for secrets-store-csi-driver?

Reply:
us-west-2 will always have the required capacity to run aws tests for secrets-store-csi-driver. It along with us-east-1 are the highest volume regions/regions with the most activity 

This account because it's shared between different teams has limits beyond default limits